app/home/homeController.js

```diff
@@ -8,7 +8,9 @@
 
     function controllerFunction (cityService) {
           var vm = this;
-          vm.city = cityService.city;
+          vm.cityService = cityService;
+          // Curious as to why you need the whole object. Commented out in forecastController.
```

Comment:
I couldn't get it to share data between controllers when only doing the city attribute =/ But when I gave it the whole object however it did work.